### ISSUE TITLE
Transaction details shows request as "Paid" while it is still confirming on the network

### ISSUE THREAD

**Issue by gary-rowe** [bitcoin-solutions/multibit-hd#516]
User @crwatkins noted that the transaction details panel view shows the payment request as "Paid" while it is still confirming on the network. This should really be "In progress" or similar until confirmed.

**Comment by jim618** [comment]
I disagree on this . Paid / Not paid / Part paid is the status of whether you have decided to pay it or not.
It's separate to the state machine of 'seen by network/ confirmed by one block/ confirmed by 3 blocks etc'

As long as the tx gets out to the network ok we should just mark it as Paid.

**Comment by crwatkins** [comment]
My original comment related to the confusion on the receiving end of seeing a payment request Status as "Paid", yet I don't see that show up in my balance (because it is not confirmed, I suspect).

**Comment by jim618** [comment]
We don't show unconfirmed payments in the balance because doing so enables a very easy social attack where someone sends you an unconfirmable tx and the balance increases.

That's why we show 'current activity' on the Send / Request screen.

Paid/ Unpaid/ Part paid = has the user sent the tx
Confirmations/ blocks = Bitcoin network stuff
